src/installer.ts

```diff
@@ -186,44 +186,43 @@ export class DotnetCoreInstaller {
     osSuffixes: string[],
     version: string
   ): Promise<string[]> {
-    let downloadUrls = [];
-    let releasesJSON = await this.getReleasesJson();
-    core.debug('Releases: ' + releasesJSON);
+    let downloadUrls: string[] = [];
 
-    let releasesInfo = JSON.parse(await releasesJSON.readBody());
+    var httpCallbackClient = new httpClient.HttpClient('setup-dotnet', [], {});
+    const releasesJsonUrl = await this.getReleasesJsonUrl(
+      httpCallbackClient,
```

Comment:
It reads weird to pass the client to a function then use the client in the next method.

Reply:
Yeah, not sure I have a better option though. I did this because I didn't want to create 2 clients, I wanted to modularize, and I need the url for the error message below.